### PR TITLE
Remove kubeadm pull images

### PR DESCRIPTION
**Pull request by priyawadhwa**
Since we pull and load all of these images already, we shouldn't need to do it again.

### CODE REVIEW

**Comment by tstromberg** [comment]
```
cmd/minikube/cmd/start.go:1299:121: `bootstrapCluster` - `preexisting` is unused (unparam)
```

**Comment by medyagh** [comment]
seems harmless ! except the lint :)

File: cmd/minikube/cmd/start.go [hunk 1]
```diff
@@ -1205,55 +1204,7 @@ func getKubernetesVersion(old *config.MachineConfig) (string, bool) {
...
-	oldestVersion, err := semver.Make(strings.TrimPrefix(constants.OldestKubernetesVersion, version.VersionPrefix))
-	if err != nil {
-		exit.WithCodeT(exit.Data, "Unable to parse oldest Kubernetes version from constants: {{.error}}", out.V{"error": err})
-	}
-	defaultVersion, err := semver.Make(strings.TrimPrefix(constants.DefaultKubernetesVersion, version.VersionPrefix))
-	if err != nil {
-		exit.WithCodeT(exit.Data, "Unable to parse default Kubernetes version from constants: {{.error}}", out.V{"error": err})
-	}
-
-	if nvs.LT(oldestVersion) {
```
Comment (tstromberg) [inline cmd/minikube/cmd/start.go]: Where'd these warnings go?

Comment (priyawadhwa) [reply cmd/minikube/cmd/start.go]: I'm trying to determine if we still need to pull images if we have a version upgrade on a preexisting cluster -- just waiting to see how `TestVersionUpgrade` responds to this.

**Pull request merged**